lib/publish.js

```diff
@@ -18,23 +22,24 @@ const npmPublish = options => {
 	return execa('npm', args);
 };
 
-const handleError = (task, err, tag, message) => {
+const handleError = (task, err, options, message) => {
 	if (err.stderr.includes('one-time pass')) {
 		const {title} = task;
 		task.title = `${title} ${chalk.yellow('(waiting for input…)')}`;
 
 		return listrInput(message || 'Enter OTP:', {
 			done: otp => {
 				task.title = title;
-				return npmPublish({tag, otp});
+				options = Object.assign({otp}, options);
+				return npmPublish(options);
```

Comment:
Just do `npmPublish(Object.assign({otp}, options))`

Reply:
applied on [this commit](https://github.com/sindresorhus/np/pull/263/commits/4bca04547126957e9330ab0ef0351dafe55fa8b2). Thanks for the review @SamVerschueren